### PR TITLE
Wraps text in preamble as well

### LINKED ISSUE #103: WordWrap the preamble, not only the argument description.
title says it all.

### PR DESCRIPTION
**Pull request by yfarjoun**
- fixes #103
- also adds some finals and other aesthetics

### CODE REVIEW

File: src/test/java/org/broadinstitute/barclay/argparser/LegacyCommandLineArgumentParserTest.java [hunk 1]
```diff
@@ -858,32 +858,66 @@ public void testHTMLConverter(String input, String expected) {
...
+
+        final String expected = "USAGE: TestParserLongPreamble [options]\n" +
+                "\n" +
+                "X < Y This is the first row but it's really long and it has lots of words...really big words. Because it knows the best\n" +
+                "people and really has lots of interesting things it needs to get across. Definitely more than can fit in 140\n" +
+                "characters...";
+
+        final String result = byteArrayOutputStream.toString();
+        Assert.assertEquals(result.substring(0, expected.length()), expected);
     }
```
[the remark this answers is not in the document]
Comment (yfarjoun) [reply src/test/java/org/broadinstitute/barclay/argparser/LegacyCommandLineArgumentParserTest.java]: hmmm. I tried doing that but the inverse html doesn't seem to work in nonLegacy...am I doing something wrong?

[the remark this answers is not in the document]
Comment (cmnbroad) [reply src/test/java/org/broadinstitute/barclay/argparser/LegacyCommandLineArgumentParserTest.java]: It looks like non legacy parser doesn't do the html->text rendering that the legacy parser does. I added a created https://github.com/broadinstitute/barclay/issues/112 for this. Hopefully that won't block adding the test.

**Comment by yfarjoun** [comment]
no, it's just that it means that the test has to be different...

File: src/test/java/org/broadinstitute/barclay/argparser/LegacyCommandLineArgumentParserTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (yfarjoun) [reply src/test/java/org/broadinstitute/barclay/argparser/LegacyCommandLineArgumentParserTest.java]: done

**Comment by yfarjoun** [comment]
back to you @cmnbroad

**Comment by yfarjoun** [comment]
bump.

**Pull request merged**